internal/generator/ecc/template/point.go.tmpl

```diff
@@ -1521,52 +1521,6 @@ func (p *{{ $TProjective }}) FromAffine(Q *{{ $TAffine }}) *{{ $TProjective }} {
 	return p
 }
 
```

Comment:
Why are we removing the method? Didn't this allow for efficient MSM?

Reply:
Projective coordinates iirc are used only for pairings (G2 for ate pairing and G1 for Tate pairing). This allows to mutualise some computations between the G1/G2 arithmetic and the line computation. But for scalar multiplication we use Jacobian coordinates as they are faster and for MSM we used extended Jacobians and affine coordinates.

Reply:
I see, thanks!